content/en/tracing/setup_overview/open_standards/_index.md

```diff
@@ -93,6 +96,196 @@ service:
       exporters: [datadog/api]
 ```
 
+### Environment specific setup
+
+#### Host
+
+1. Download the appropriate binary from [the project repository latest release][11].
+
+2. Create a `otel_collector_config.yaml` file. [Here is an example template](#ingesting-opentelemetry-traces-with-the-collector) to get started. It enables the collector's OTLP Receiver and Datadog Exporter.
+
+3. Run on the host with the configration yaml file set via the `--config` parameter. For example,
+
+  ```
+    otelcontribcol_linux_amd64 --config otel_collector_config.yaml
+  ```
+
+#### Docker
+
+Run an Opentelemetry Collector container to receive traces either from the [installed host](#receive-traces-from-host), or from [other containers](#receive-traces-from-other-containers).
+
+##### Receive Traces from host
+
+1. Create a `otel_collector_config.yaml` file. [Here is an example template](#ingesting-opentelemetry-traces-with-the-collector) to get started. It enables the collector's OTLP receiver and the Datadog exporter.
+
+2. Choose a published docker image such as [`otel/opentelemetry-collector-contrib:latest`][12].
+
+3.  Determine which ports to open on your container. OpenTelemetry Traces are sent to the OpenTelemetry Collector over TCP or UDP on a number of ports, which must be exposed on the container.  By default traces are sent over OTLP/gRPC on port `55680`, but common protocols and their ports include: 
+
+  - Zipkin/HTTP on port `9411`
+  - Jaeger/gRPC on port `14250`
+  - Jaeger/HTTP on port `14268`
+  - Jaeger/Compact on port (UDP) `6831` 
+  - OTLP/gRPC on port `55680`
+  - OTLP/HTTP on port `55681`
+
+4. Run the container with the configured ports and an `otel_collector_config.yaml` file. For example:
+
+  ```
+   $ docker run \
+    -p 55680:55680 \
+    -v $(pwd)/otel_collector_config.yaml:/etc/otel/config.yaml \
+    otel/opentelemetry-collector-contrib:latest
```

Comment:
```suggestion
    otel/opentelemetry-collector-contrib
```

Reply:
if we change this, do we also change line 121?

Reply:
Yes, good catch!